modules/images/image-loading-optimization/load.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Module Name: Image Loading Optimization
+ * Description: Improves accuracy of optimizing the loading of the LCP image by leveraging client-side detection with real user metrics. Also enables output buffering of template rendering which can be filtered.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+// Do not load the code if it is already loaded through another means.
+if ( function_exists( 'image_loading_optimization_buffer_output' ) ) {
+	return;
+}
```

Comment:
Can you replace this with an `IMAGE_LOADING_OPTIMIZATION_VERSION` constant, similar to the other modules?

Those should be used on every module for interoperability with standalone plugins. The `function_exists()` checks were added before those constants were required, but because of the constants they are now no longer needed (we could even remove them from the other modules, but not important).